### PR TITLE
Add tool dependency definition for datamash.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: packages/package_datamash_1_0_6/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="datamash" version="1.0.6">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url">http://ftp.gnu.org/gnu/datamash/datamash-1.0.6.tar.gz</action>
```
Comment (erasche) [inline packages/package_datamash_1_0_6/tool_dependencies.xml]: sha256sum please

File: packages/package_datamash_1_0_6/tool_dependencies.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                <action type="shell_command">./configure --prefix=$INSTALL_DIR --disable-dependency-tracking</action>
+                <action type="shell_command">make</action>
+                <action type="shell_command">make check</action>
+                <action type="shell_command">make install</action>
```
Comment (erasche) [inline packages/package_datamash_1_0_6/tool_dependencies.xml]: if you can get away without `make check`, autoconf might be an alternative here.

**Comment by erasche** [comment]
LGTM.

File: packages/package_datamash_1_0_6/tool_dependencies.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="datamash" version="1.0.6">
+        <install version="1.0">
+            <actions>
+                <action sha256sum="0154c25c45b5506b6d618ca8e18d0ef093dac47946ac0df464fb21e77b504118"
+                    type="download_by_url">http://ftp.gnu.org/gnu/datamash/datamash-1.0.6.tar.gz</action>
+                <action type="autoconf">--disable-dependency-tracking</action>
+                <action type="make_install" />
```
Comment (bgruening) [inline packages/package_datamash_1_0_6/tool_dependencies.xml]: Isn't `make_install` redundant? `autoconf` is doing the configure, make, make install dance.

Comment (erasche) [inline packages/package_datamash_1_0_6/tool_dependencies.xml]: `:|` yep. I coulda sworn that was red earlier. Will get this patched.

**Pull request merged**